src/app/components/buscadorTurnosPrestaciones/turnos-prestaciones.scss

```
@@ -1,4 +1,21 @@
 table th.sortable:hover {
     text-decoration: underline !important;
     cursor: pointer;
-}
\ No newline at end of file
+}
+
+cdk-virtual-scroll-viewport {
+    height: 100%;
```

Para evitar el doble scroll:

```suggestion
    height: calc(100% - 180px);;
```

---

Hecho